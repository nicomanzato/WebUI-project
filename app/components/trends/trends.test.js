import React from 'react';
import Trends from './trends';
import TrendMock from './../../store/trend/mock/trendMock'
import renderer from 'react-test-renderer';

describe('trends', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <Trends
          trends={[TrendMock, TrendMock, TrendMock]}
          onTrendsPress={() => {}}
        />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
