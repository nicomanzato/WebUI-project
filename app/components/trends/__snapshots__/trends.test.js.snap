// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`trends should render correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 15,
        "marginLeft": 10,
      }
    }
  >
    Trends for you
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "#dddddd",
        "height": 1,
      }
    }
  />
  <RCTScrollView>
    <View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={null}
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "borderBottomWidth": 1,
              "borderColor": "#ccc",
              "flexDirection": "row",
              "padding": 10,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 0.1,
                "paddingLeft": 7,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "grey",
                  "fontSize": 18,
                }
              }
            >
              1
            </Text>
          </View>
          <View
            style={
              Object {
                "flex": 0.9,
                "paddingLeft": 10,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "fontSize": 17,
                  "marginBottom": 5,
                }
              }
            >
              Sofía Herrera
            </Text>
          </View>
        </View>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={null}
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "borderBottomWidth": 1,
              "borderColor": "#ccc",
              "flexDirection": "row",
              "padding": 10,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 0.1,
                "paddingLeft": 7,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "grey",
                  "fontSize": 18,
                }
              }
            >
              2
            </Text>
          </View>
          <View
            style={
              Object {
                "flex": 0.9,
                "paddingLeft": 10,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "fontSize": 17,
                  "marginBottom": 5,
                }
              }
            >
              Sofía Herrera
            </Text>
          </View>
        </View>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={null}
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Object {
              "borderBottomWidth": 1,
              "borderColor": "#ccc",
              "flexDirection": "row",
              "padding": 10,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 0.1,
                "paddingLeft": 7,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "grey",
                  "fontSize": 18,
                }
              }
            >
              3
            </Text>
          </View>
          <View
            style={
              Object {
                "flex": 0.9,
                "paddingLeft": 10,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "fontSize": 17,
                  "marginBottom": 5,
                }
              }
            >
              Sofía Herrera
            </Text>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
